src/tests/axiosConfig.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
```

Comment:
I think you already defined this module in `src/axiosConfig.js` so I don't see why you should do it again here. Unless you have a compelling reason, I'd advise you do away with this